api/read.yaml

```diff
@@ -0,0 +1,97 @@
+openapi: 3.0.1
+info:
+  title: Ampersand public read API
+  version: 1.0.0
+servers:
+  - url: https://read.withampersand.com/v1
+paths:
+  /projects/{projectIdOrName}/integrations/{integrationId}/objects/{objectName}:
+    post:
+      summary: Create read operation for records
+      operationId: createRead
+      tags: ["Read"]
+      parameters:
+        - name: projectIdOrName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: integrationId
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: objectName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: X-Amp-Request-Type
```

Comment:
X-Amp-Request-Type is added by our kong server, the user doesn't have to know about it, please remove.